src/components/footer/footer.tsx

```diff
@@ -57,11 +57,15 @@ const footerLinks = [
 ]
 
 export const Footer = () => {
+    const scrollToTop = () => {
+        window.scrollTo({ top: 0, left: 0, behavior: 'smooth' });
+    };
+
     return (
       <footer className="footer__root">
         <Container maxWidth="md" id="footer-container">
             <Box className="footer__box">
-                <div className="footer__logo">
+                <div onClick={scrollToTop} className="footer__logo">
```

Comment:
NIT
I usually prefer the arrow function syntax for onclick like below.

```suggestion
                <div onClick={() => scrollToTop()} className="footer__logo">
```

There are several articles like [this one](https://blog.devgenius.io/why-it-is-necessary-to-use-arrow-functions-with-react-event-handler-e0b278710310) that highlights the difference and advantage of using arrow functions in event handlers. Otherwise, great job

Reply:
Understood! Thanks for sharing the article!